_includes/tutorials/multi-joins/ksql/code/Makefile

```
@@ -0,0 +1,16 @@
+STEPS_DIR := tutorial-steps
+DEV_OUTPUTS_DIR := $(STEPS_DIR)/dev/outputs
+TEST_OUTPUTS_DIR := $(STEPS_DIR)/test/outputs
+PROD_OUTPUTS_DIR := $(STEPS_DIR)/prod/outputs
+TEMP_DIR := $(shell mktemp -d)
+SEQUENCE := "dev, test, prod, ccloud"
+
+tutorial:
+	rm -r $(DEV_OUTPUTS_DIR) || true
+	rm -r $(TEST_OUTPUTS_DIR) || true
+	mkdir $(DEV_OUTPUTS_DIR)
+	mkdir -p $(TEST_OUTPUTS_DIR)
+	harness-runner ../../../../../_data/harnesses/multi-joins/ksql.yml $(TEMP_DIR) $(SEQUENCE)
+	diff --strip-trailing-cr $(STEPS_DIR)/dev/expected-transient-query.log $(DEV_OUTPUTS_DIR)/transient-query/output-0.log
+	bash -c "diff --strip-trailing-cr <(sort <(cut -d ',' -f 2- $(STEPS_DIR)/dev/expected-print-topic.log)) <(sort <(cut -d ',' -f 2- $(DEV_OUTPUTS_DIR)/print-topic/output-0.log))"
```

```suggestion
	bash -c "diff --strip-trailing-cr <(sort <(cut -d ',' -f 2- $(STEPS_DIR)/dev/expected-print-topic.log)) <(sort <(cut -d ',' -f 2- $(DEV_OUTPUTS_DIR)/print-topic/output-0.log))"
	bash -c "diff --strip-trailing-cr <(sort $(STEPS_DIR)/test/expected-results.log) <(sort $(TEST_OUTPUTS_DIR)/test-results.log)"
```
Would you mind adding a check against the testing section of this tutorial? (might need to double check that this is the same expected test + test results log file name and location)

---

Thanks @awalther28 , I added your suggestion and verified the tests pass.  Thank you